packages/core/src/oidc/adapter.ts

```diff
@@ -68,10 +66,7 @@ export default function postgresAdapter(modelName: string): ReturnType<AdapterFa
       client_id,
       client_secret,
       client_name,
-      application_type: getApplicationTypeString(type),
-      grant_types: Object.values(GrantType),
-      token_endpoint_auth_method:
-        type === ApplicationType.Traditional ? 'client_secret_basic' : 'none',
+      ...getConstantClientMetadata(type),
```

Comment:
https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#clients

Can't recall. Is there any specific response we did not put `application_type`, `grant_types` and `token_endpoint_auth_method` in DB as part of the OIDC client metadata?

Reply:
We want to hard-code them to fit the best practice. So I think no need to store in DB